### PR TITLE
mu4e: update to 1.8.14

### PR DESCRIPTION
**Pull request by pascal-huber**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc
- I built this PR locally for these architectures:
  - x86_64-musl (crossbuild)
  - armv7l (crossbuild)

### CODE REVIEW

File: srcpkgs/mu4e/template [hunk 1]
```diff
@@ -11,7 +11,7 @@ maintainer="Benjamin Slade <slade@lambda-y.net>"
 license="GPL-3.0-or-later"
 homepage="https://www.djcbsoftware.nl/code/mu/"
```
Comment (classabbyamp) [inline srcpkgs/mu4e/template:12]:
```suggestion
homepage="https://www.djcbsoftware.nl/code/mu/"
changelog="https://github.com/djcb/mu/raw/master/NEWS.org"
```

Comment (pascal-huber) [reply srcpkgs/mu4e/template:12]: I intentionally didn't add this as it doesn't reflect changes of minor versions. The changelog seems to be partially in `News.org` and in the description of the releases on github. Still want me to add it?

Comment (classabbyamp) [reply srcpkgs/mu4e/template:12]: i think it's still good to add

Comment (pascal-huber) [reply srcpkgs/mu4e/template:12]: done.

**Pull request merged**